### PR TITLE
Update dist and add CI task to fail if dist isn't committed

### PR DESCRIPTION
**Pull request by tvararu**
Also add contributing instructions to README, and add a check in CI that will throw if someone tries to merge in code that hasn't committed the dist.

### CODE REVIEW

File: src/dfe-autocomplete.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ const suggestion = (value, options) => {
     const html = option.append ? `<span>${value}</span> ${option.append}` : `<span>${value}</span>`
     return option.hint ? `${html}<br>${option.hint}` : html
   } else {
-    return `<span>No results found</span>`
+    return `<span>No results foundd</span>`
```
Comment (tomas-stefano) [inline src/dfe-autocomplete.js:20]: Typo

**Review by tomas-stefano — APPROVED** [review]
LG. Minor thing just fixing typo and it is good to be merged

File: src/dfe-autocomplete.js (hunk 1 above)
Comment (tvararu) [reply src/dfe-autocomplete.js]: It was a temporary change, I was testing to see if CI actually fails

Comment (tomas-stefano) [reply src/dfe-autocomplete.js]: Awesome work so it is all good

**Pull request merged**